amethyst_network/src/bundle.rs

```diff
@@ -34,10 +38,10 @@ impl<'a, 'b, 'c, T> ECSBundle<'a, 'b> for NetworkClientBundle<'c,T> {
         world.add_resource(NetReceiveBuffer::<T>::new());
 
         while self.port.is_none() || self.port.unwrap() < 200 {
-            self.port = Some(rand::random::<u32>());
+            self.port = Some(rand::random::<u16>());
```

Comment:
Should notify the user that configured `port < 200` and it is not going to happen.
Not sure why `200`. Dynamic ports are `49152 .. 65535`.
Also you need to be sure that chosen port is not used.

Reply:
Below 2000 instead of 200 yes?  Below 2000 require root access.

Reply:
1024 for some OSs, but i concur, > 2000 should be fine.

Reply:
There should be an API function somewhere that will give you an unprivileged random port that is unused. Trying with random is gonna be flaky.

Reply:
fixed